### DISCUSSION TITLE
inference masks

### DISCUSSION THREAD

**Discussion by GOfBlue** [anthonyweidai/SvANet#17] [Q&A] [answered]
During inference, only files such as "tiny metrics avg.csv" will be generated. Is there any relevant code for generating inference masks that can be provided? There is a display of inference masks in the paper. I am very grateful to the author for your work.Hope you have more achievements！

**Comment by anthonyweidai** [comment, marked as answer]
Please refer to https://pytorch.org/vision/main/auto_examples/others/plot_visualization_utils.html.

**Comment by Zheng912** [comment]
> During inference, only files such as "tiny metrics avg.csv" will be generated. Is there any relevant code for generating inference masks that can be provided? There is a display of inference masks in the paper. I am very grateful to the author for your work.Hope you have more achievements！

Hello! Have you resolved this issue? I referred to the link, but unfortunately, I couldn't generate the mask images as shown in the paper. Could you please share the code you used to generate the mask files? Thank you so much!
